backend/easymed/celery_tasks.py

```diff
@@ -256,6 +259,15 @@ def check_inventory_reorder_levels():
     Periodically checks all inventory items for reorder levels and sends notifications if needed.
     """
     items = Inventory.objects.filter(quantity_at_hand__lte=F('re_order_level'))
+    User = get_user_model()
+
+    users_to_notify = User.objects.filter(role__in=[User.SYS_ADMIN, User.LAB_TECH])
```

Comment:
Let's pass in list of users to notify from fronted. It's easier to let the hospital customize the list. Suggestion: You can create an endpoint with a model that will determine which users get notified. If the hospital wants someone "unconventional" to get these notifications, we will have to jump into the code and roll out updates.  

Another suggestion: We can go the permissions route. A user can belong to a Group: "Say INVENTORY NOTIFICATIONS" with "CAN_GET_INVENTORY_NOTIFICATIONS" permissions to it.